python_modules/libraries/dagster-dg/dagster_dg_tests/utils.py

```diff
@@ -638,7 +642,7 @@ def invoke(self, *args: str, **invoke_kwargs: Any) -> Result:
         # We need to find the right spot to inject global options. For the `dg scaffold`
         # command, we need to inject the global options before the final subcommand. For everything
         # else they can be appended at the end of the options.
```

Comment:
can we update this comment with a "why" component since i had to ask

Reply:
yes I'll add it